uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -46,6 +47,31 @@ var Promise = require('es6-promise').Promise;
     });
   });
 
+  /**
+   * Gets a map of contact roles and IDs
+   *
+   * @return {Promise} resolved with {Object}
+   */
+  function getRolesAndIDs () {
+    return new Promise(function (resolve) {
+      var rolesAndIDs = {};
+
+      exec('cv api contact.get', {maxBuffer: 1024 * 1000}, function (err, result) {
+        if (err) {
+          console.log(color('Unable to fetch contact roles and IDs: ' + err, 'RED'));
+        }
+
+        _.each(JSON.parse(result).values, function (contact) {
+          if (contact.email.match(/^civihr_/)) {
+            rolesAndIDs[contact.email.replace(/^civihr_([^@]+)@.*$/, '$1')] = contact.id;
+          }
+        });
+
```

Comment:
In the unlikely case of not having any emails with `civihr`, we are not throwing any error, so the tests would still start failing, but we wont know whats ging wrong. Can we add a console message here with warning.

Reply:
![image](https://user-images.githubusercontent.com/3973243/35102103-d2b283ee-fc59-11e7-8ed4-964d52ce0744.png)